addons/bundle-analyzer/README.md

```diff
@@ -0,0 +1,3 @@
+# Webpack Bundle Analyzer for Storybook
+
+Exposes Webpack Bundle Analyzer results in a Storybook addon panel.
```

Comment:
What do you think of adding an explanation like this? So that people don't necessarily need to find the webpack bundle analyzer website to understand it better. I got the text from their repo.

```suggestion
Exposes Webpack Bundle Analyzer results in a Storybook addon panel.

This addon will help you:

- Realize what's really inside your application bundle
- Find out what modules make up the most of its size
- Find modules that got there by mistake
- Optimize it!

```